Make return statement optional (#1222)
Zokrates/ZoKrates: zokrates_book/src/language/functions.md

```zokrates
{{#include ../../../zokrates_cli/examples/book/explicit_generic_parameters.zok}}
```

> Functions that "don't" return a value, actually return an empty tuple `()`. 
> When a function returns an empty tuple `()`, the return type can be omitted from 
> the signature. In this case, a return statement can be omitted.

Reviewer: ```suggestion
If the return type of a function is the empty tuple `()`, the return type as well as the return statement can be omitted.
```

Then link to code like

```
def main() {
}
```
